pipeline_tools/get_upload_urn.py

```diff
@@ -35,18 +35,18 @@ def urn_is_none(response):
             retry=retry_if_result(urn_is_none)
         )
     )
-    urn = get_staging_urn(response.json())
+    urn = get_upload_urn(response.json())
     return urn
 
 
-def get_staging_urn(envelope_js):
-    """Get the staging urn from the submission envelope.
+def get_upload_urn(envelope_js):
+    """Get the upload urn from the submission envelope.
 
     Args:
         envelope_js (dict): the submission envelope contents
 
     Returns:
-        String giving the staging urn in the format dcp:upl:aws:integration:12345:abcde,
+        String giving the upload urn in the format dcp:upl:aws:integration:12345:abcde,
```

Comment:
nit: change from `dcp:upl:<stage>:<uuid>:<encoded-credentials>` to `s3://<bucket>/<uuid>`